.github/workflows/ci-workflow-pull-request.yml

```
@@ -211,6 +211,16 @@ jobs:
       contents: read
     uses: ./.github/workflows/build-matx.yml
 
+  build-stdpar:
+    name: Build NVHPC stdpar
+    secrets: inherit
+    permissions:
+      id-token: write
+      contents: read
+    uses: ./.github/workflows/build-stdpar.yml
+    with:
+      enable_slack_alerts: true
```

We don't want this for PRs.

```suggestion
      enable_slack_alerts: false
```

---

yeah I want to move this to a different place so that it runs similar to our CUB / thrust / libcu++ CI